reV/pipeline/pipeline.py

```diff
@@ -354,10 +343,10 @@ def _get_cmd(cls, command, f_config, verbose=False):
         cmd : str
             Python reV CLI call string.
         """
-        if command not in cls.COMMANDS:
-            raise KeyError('Could not recongize command "{}". '
-                           'Available commands are: {}'
-                           .format(command, cls.COMMANDS))
+        if command not in ModuleName.all_names():
+            raise KeyError("Could not recognize command {!r}. "
+                           "Available commands are: {!r}"
+                           .format(command, ModuleName.all_names())) from None
```

Comment:
Can we actually define `cls.COMMANDS=ModuleName.all_names()` to maintain the class attribute functionality? We use this in NSRDB.

Reply:
Yup! I will make that change right now.